sql/hive/src/main/scala/org/apache/spark/sql/hive/TableReader.scala

```diff
@@ -111,7 +111,8 @@ class HadoopTableReader(
       filterOpt: Option[PathFilter]): RDD[InternalRow] = {
 
     assert(!hiveTable.isPartitioned, """makeRDDForTable() cannot be called on a partitioned table,
-      since input formats may differ across partitions. Use makeRDDForTablePartitions() instead.""")
+      since input formats may differ across partitions. Use makeRDDForPartitionedTable() 
```

Comment:
I think it's gonna keep the white spaces as is. Mind if I ask change this as below?

```
assert(
  !hiveTable.isPartitioned,
  "makeRDDForTable() cannot be called on a partitioned table, since input formats may differ " +
    "across partitions. Use makeRDDForTablePartitions() instead."
```

Reply:
That's Good. I will modify it, thanks.